OTEL Remote Management GA (#4875)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-collector/remote-management/overview.md

---
id: overview
title: OpenTelemetry Remote Management Overview
sidebar_label: Overview
---

import useBaseUrl from '@docusaurus/useBaseUrl';

The Sumo Logic Distribution for OpenTelemetry Collector simplifies remote management of data collection, allowing setup from the Sumo Logic UI and deployment to multiple collectors.

> ## Key benefits of Remote Management

Reviewer: ```suggestion
## Key benefits of remote management
```

> ## Key benefits of Remote Management

Reviewer: I see "Remote Management" and "remote management" used in this article. I think you should change them all to "remote management".

* Simplified setup and configuration via the Sumo Logic UI.
* Tag-based collector grouping for efficient data collection.

> * Centralized configuration using [Source Templates](/docs/send-data/opentelemetry-collector/remote-management/source-templates).

Reviewer: ```suggestion
* Centralized configuration using [source templates](/docs/send-data/opentelemetry-collector/remote-management/source-templates).
```

* No server access required after installation.
* Faster time to value and reduced manual errors.

## Common use cases

* Monitoring error logs across multiple Apache servers.

> * Grouping collectors by environment (e.g., production, staging).

Reviewer: ```suggestion
* Grouping collectors by environment (for example, production, staging).
```

1. After installation is complete, click **Next** to proceed.
1. On the next screen, you will see a list of available Source Templates. Select the **Apache Source Template** to apply the source template to start collecting logs from all linked collectors.

To revisit this screen later, navigate to:
* [**Classic UI**](/docs/get-started/sumo-logic-ui-classic): In the main Sumo Logic menu, select **Manage Data > Collection > Source Template**.
* [**New UI**](/docs/get-started/sumo-logic-ui): In the Sumo Logic top menu select **Configuration**, and then under **Data Collection**, select **Source Template**.  

### Step 2: Configure data collection

Next, you'll create a data collection configuration to gather Apache error logs and link it to all collectors tagged `"application = Apache"`.

1. Complete the Source Template form with the **Name** and **File Path** for your error logs, then click **Next**.

> 1. Under **Link Collectors**, add the tag `"application = Apache"`.<br/><img src={useBaseUrl('img/send-data/local-file-apache.png')} alt="Screenshot of the file path configuration for Apache logs" width="300"/>

Reviewer: ```suggestion
1. Under **Link Collectors**, add the tag `"application = Apache"`.<br/><img src={useBaseUrl('img/send-data/local-file-apache.png')} alt="Screenshot of the file path configuration for Apache logs" style={{border: '1px solid gray'}} width="300"/>
```
